helm/ocs-monkey-generator/values.yaml

```diff
@@ -15,6 +15,23 @@ osio:
   kernelUntar: 6
   kernelRm: 6
 
+# Set this parameter to install chaos runner
+runChaosRunner: false
+
+# Name of chaos runner deployment
+chaosName: ocs-monkey-chaos
+
+# Parameters to configure chaos runner
+chaos:
+  additionalFailure: 0.25
+  checkInterval: 30
+  cephclusterName: ocs-storagecluster-cephcluster
+  logDir: /logs
+  mitigationTimeout: 900
+  mttf: 150
+  ocsNamespace: openshift-storage
+  monitorDeployment: openshift-storage/ocs-monkey-wr-ocs-monkey-generator
```

Comment:
This is unused

Reply:
Only ocsNamespace and monitorDeployment, right?